src/Command/DecodeCommand.php

```diff
@@ -12,10 +12,12 @@
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use UnexpectedValueException;
 
 #[AsCommand(
     name: 'smile-decode',
-    description: 'Decodes a smile binary file. You may directly decode a string as well.'
+    description: 'Decodes a smile binary file as json. You may directly decode a smile string as well.'
```

Comment:
```suggestion
    description: 'Decodes a smile binary file as JSON. You may directly decode a Smile string as well.'
```

acronyme must be uppercase